packages/taquito-rpc/test/taquito-rpc.spec.ts

```diff
@@ -220,20 +220,6 @@ describe('RpcClient test', () => {
     });
   });
 
-  describe('getBigMapKey', () => {
-    it('query the right url', async done => {
-      await client.getBigMapKey('address', { key: 'test', type: 'string' } as any);
-      expect(httpBackend.createRequest.mock.calls[0][0]).toEqual({
-        method: 'POST',
-        url: 'root/chains/test/blocks/head/context/contracts/address/big_map_get',
-      });
-
-      expect(httpBackend.createRequest.mock.calls[0][1]).toEqual({ key: 'test', type: 'string' });
-
-      done();
-    });
-  });
-
```

Comment:
We need to keep the test coverage despite the function being deprecated. Can you add this test back, and use a lint hint to tell the linter to ignore the use of the deprecated function.

Reply:
On it